vnext/ReactUWP/Views/TextViewManager.cpp

```diff
@@ -179,5 +184,23 @@ YGMeasureFunc TextViewManager::GetYogaCustomMeasureFunc() const {
   return DefaultYogaSelfMeasureFunc;
 }
 
+void TextViewManager::OnDescendantTextPropertyChanged(ShadowNodeBase *node) {
```

Comment:
>OnDescendantTextPropertyChanged [](start = 22, length = 31)

Does the change work on Nested text?

Reply:
yes. it notified all TextView parents, and the logic is in NotifyAncestorsTextChanged